documentation/src/main/wiki/UpgradeNotes.md

```diff
@@ -7,3 +7,29 @@ You may see all changes in the [Changelog](https://github.com/Cognifide/aet/blob
 
 ## Unreleased
 
+### [PR-326](https://github.com/Cognifide/aet/pull/326) Upgrade OSGI annotations to 6.0.0 version
+
+With the update osgi annotations to 6.0.0 version we had to change a little bit variable names. Currently, your config could has old names and you have to update them. Please follow instruction below:
```

Comment:
Please fix grammar (correct me if I'm wrong, but I think it should be like this): 
`config could has` -> `config could have`
`With the update osgi annotations to 6.0.0` -> `With the OSGI annotations update to 6.0.0`

Reply:
Done